package.json

```diff
@@ -49,7 +49,8 @@
     "xtend": "^4.0.1"
   },
   "peerDependencies": {
-    "react": "^15.0.0 || ^16.0.0"
+    "react": "^15.0.0 || ^16.0.0",
+    "@types/react": "^15.0.0 || ^16.0.0"
```

Comment:
btw, isn’t 17 around the corner? Could we allow that? (e.g., `>= 15.0.0` or so)?

Reply:
It is, see https://github.com/remarkjs/react-markdown/pull/490#discussion_r506969473

We could add it :thinking:
I haven't tested it yet to ensure it still works.
May hope is it should be painless transition, that will only require an update to the peer dependencies.

Reply:
could be added later!